packages/records/lib/models/records.integration.test.ts

```diff
@@ -218,6 +218,41 @@ describe('Records service', () => {
                 });
             });
         });
+
```

Comment:
Just moving this test to be inside of the upsert section

Reply:
it's easier to review with "hide whitespace"